Samples/Cordova/ExampleProject/platforms/ios/ExampleProject/ExampleProject-Info.plist

```diff
@@ -50,11 +50,11 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
-	<key>CleverTapRegion</key>
-	<string></string>
 	<key>CleverTapAccountID</key>
 	<string>TEST-R78-ZZK-955Z</string>
 	<key>CleverTapToken</key>
 	<string>TEST-311-ba2</string>
+	<key>CleverTapRegion</key>
```

Comment:
@AishwaryaNanna Why is this key empty?

Reply:
Its updated with latest ios sdk version update

Reply:
removed the optional region key.